projects/botan/build.sh

```diff
@@ -17,20 +17,17 @@
 
 cd $SRC/botan
 
+ln -s $SRC/fuzzer_corpus .
+
 # This assumes $CC is set to either 'clang' or 'gcc'
 ./configure.py --cc=$CC --cc-bin=$CXX --cc-abi-flags="$CXXFLAGS" \
-               --unsafe-fuzzer-mode --disable-shared --disable-modules=locking_allocator
-make -j$(nproc) libbotan-2.a
-
-jigs=$(find $SRC/botan/src/extra_tests/fuzzers/jigs -name "*.cpp")
-
-for fuzzer_src in $jigs; do
-  fuzzer=$(basename $fuzzer_src .cpp)
-  $CXX $CXXFLAGS -DUSE_LLVM_FUZZER -std=c++11 -I$SRC/botan/build/include \
-       -o $OUT/$fuzzer $fuzzer_src -L$SRC/botan -lbotan-2 -lFuzzingEngine
+               --disable-shared --disable-modules=locking_allocator \
+               --unsafe-fuzzer-mode --build-fuzzers=libfuzzer \
```

Comment:
is `--build-fuzzers=libfuzzer` a problem if we are building with other fuzzing engines? (e.g. afl, honggfuzz)

Reply:
No as far as the build is concerned this just means "uses the libfuzzer interface" (`LLVMFuzzerRunTestOneInput`) vs the AFL interface (main + __AFL_LOOP).